lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -155,7 +155,23 @@ export default async ({
     },
     optimization: {
       splitChunks: {
-        chunks: 'all',
+        cacheGroups: {
+          prettier: {
+            test: /[\\/]node_modules[\\/]prettier[\\/]/,
+            chunks: 'async',
+            filename: '[name].bundle.js',
```

Comment:
Is it possible to do this in the DLL as well? Or completely move this to the DLL?